sdk/storage/Azure.Storage.Blobs/src/BlobContainerClient.cs

```diff
@@ -424,13 +424,14 @@ protected internal virtual BlobBaseClient GetBlobBaseClientCore(string blobName)
 
             return new BlobBaseClient(
                 blobUriBuilder.ToUri(),
-                _pipeline,
-                SharedKeyCredential,
-                Version,
-                ClientDiagnostics,
-                CustomerProvidedKey,
-                ClientSideEncryption,
-                EncryptionScope);
+                new BlobStorageContext(
```

Comment:
In the next PR, I will add `BlobStorageContext` to `BlobContainerClient`, and remove all these.